daemon/main.go

```diff
@@ -341,6 +341,7 @@ func init() {
 	flags.BoolVar(&enableTracing,
 		"enable-tracing", false, "Enable tracing while determining policy (debugging)")
 	flags.String("envoy-log", "", "Path to a separate Envoy log file, if any")
+	flags.String("403-msg", "", "Message returned in 403 body")
```

Comment:
Should we mark this as hidden so it will be an "easter-egg"? :)

Reply:
Good point, will do!